packages/dd-trace/test/appsec/index.spec.js

```diff
@@ -320,7 +324,7 @@ describe('AppSec Index', () => {
       web.root.returns(rootSpan)
     })
 
-    it('should propagate incoming http end data', () => {
+    it('should not propagate incoming http end data without express', () => {
```

Comment:
Since we removed status code and headers, the only addresses remaining are the one we extract from express response object. So "without express" means without the express fields:
![Uploading image.png…]()